src/styles/stars.scss

```
@@ -0,0 +1,32 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-inline: $margin-inline;
+
+  &_stars-block {
+    display: flex;
+    flex-direction: row;
+  }
+
+  &_star {
```

and move inside `.stars` (selectors nesting)
```suggestion
  &__star {
```

---

ok